### ISSUE TITLE
`@stylistic/declaration-block-trailing-semicolon` takes the semicolon into an inline comment under `postcss-less`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#232]
Found by the control oracle written for the plan of 2026-08-20, which lints every fixture twice — once with a `//` comment and once with a block comment of exactly the same width — and keeps the pairs whose warnings disagree.

The same question as [#117](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/117), [#207](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/207) and [#211](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/211), in the one rule of the family that asks it about a bodiless at-rule and about nothing else.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
	color: pink // c
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-less",
  "rules": {
    "@stylistic/declaration-block-trailing-semicolon": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json --fix
```

### What did you expect to happen?

The problem reported and the code left as it is, as [#117](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/117) has it for the semicolon of `declaration-block-semicolon-space-before`.

### What actually happened?

The semicolon is written onto the comment's line and commented out, and nothing is reported:

```less
a {
	color: pink // c;
}
```

### What is the cause?

`postcss-less` keeps a `//` comment standing behind a value inside that value, so the declaration is `parent.last`, the rule measures it, and the fix sets `parent.raws.semicolon = true`, which puts the semicolon immediately behind the declaration's text.

The guard the rule carries covers a bodiless at-rule and only that: it asks `endsWithInlineComment` about `raws.between`, which is the text a semicolon written behind such an at-rule would follow. A declaration wants the same question asked about its value and the raw of its `!important` together, the way [#207](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/207) asks it.

Under `postcss-scss` the same file is left alone — but by accident, and the accident is [#217](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/217): that syntax makes the comment a node of its own, so the declaration is not `parent.last` and the rule passes over it. **Fixing #217 makes the `postcss-scss` half of this reachable too**, so the two want fixing together.
